lib/webauthn/authenticator_data.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 require "webauthn/authenticator_data/attested_credential_data"
+require "webauthn/refinements/unpack1"
 
 module WebAuthn
   class AuthenticatorData
+    using WebAuthn::Refinements::Unpack1 unless String.method_defined?(:unpack1)
```

Comment:
While this is very smart, I think I slightly prefer the non-monkey-patched option of just replacing `unpack1` with `unpack("...").first`.